phys2bids/io.py

```diff
@@ -69,18 +68,22 @@ def check_multifreq(timeseries, freq, start=0, leftout=0):
     return mfreq
 
 
-def process_labchart(channel_list, chtrig, header=[]):
+def process_blueprint_items(channel_list, chtrig, interval, orig_units, orig_names):
     """
-    Process labchart header and channel_list and make a physio_obj.BlueprintInput.
+    Process items header items and channel_list and make a physio_obj.BlueprintInput.
```

Comment:
I'm not sure what you mean by `items header items and channel_list`

Reply:
so channel_list is the first parameter of the function and the header items are the rest of them

Reply:
Okay, then we should probably make the docstring clearer. It has to be descriptive for future developers too.